lib/recurly/client.php

```diff
@@ -48,7 +48,7 @@ class Recurly_Client
   const PATH_ADJUSTMENTS = '/adjustments';
   const PATH_BILLING_INFO = '/billing_info';
   const PATH_COUPON = '/coupon';
-  const PATH_COUPON_REDEMPTION = '/redemption';
+  const PATH_COUPON_REDEMPTION = '/redemptions';
```

Comment:
Unfortunately changing this breaks the [account redemption lookup call](https://docs.recurly.com/api/coupons/coupon-redemption#lookup-redemption-account) which has no specs around it. I'm not really in love with the way we're using constants here but I think we'll need to add a new `PATH_COUPON_REDEMPTIONS` constant and reference that from your changes.